src/main/kotlin/no/skatteetaten/aurora/boober/service/AuroraDeploymentContextService.kt

```diff
@@ -163,25 +164,26 @@ class AuroraDeploymentContextService(
         val allHandlers: Set<AuroraConfigFieldHandler> =
             featureHandlers.flatMap { it.value }.toSet().addIfNotNull(headerHandlers)
 
-        val applicationDeploymentId = headerSpec.run {
+        val applicationDeploymentId = headerSpec.let {
```

Comment:
Er det noen forskjell i oppførsel mellom run og let her? eller er let riktigere i forhold til bruken?

Reply:
det er helt likt. Det er bare semantikk. Ved run så vil `this` referere til `it`.
Jeg har bare forandret dette i forbifarten siden jeg personlig synes den ble litt vanskelig å lese, men kan godt bytte tilbake.

Reply:
Trenger ikke bytte tilbake